components/global/Header/NavItemWithChildren.js

```diff
@@ -9,12 +9,15 @@ import internalLinkShape, { internalLinkInternalShape } from "@/shapes/link";
 export default function NavItemWithChildren({
   id,
   active,
+  parentActive,
```

Comment:
Introduced a sense of "is my parent nav item active?" in order to collapse sibling 3rd level accordions